addons/missionTest/functions/fnc_onMissionSave.sqf

```diff
@@ -10,17 +13,36 @@ INFO_2("Placed on mission: [Units: %1] [Non-unit Objects: %2]", count _allUnits,
 
 private _bwmfDate = getText (missionConfigFile >> "bwmfDate");
 if (_bwmfDate == "") exitWith {
-    systemChat "Non-framework mission (missing description.ext?)";
+    ["Saving: Non-framework mission (missing description.ext?)"] call BIS_fnc_3DENNotification;
     WARNING("Non-framework mission (missing description.ext?)");
 };
 if (!(missionNamespace getVariable [QEGVAR(assignGear,usePotato), false])) exitWith {
-    systemChat "Potato Gear is off (missing description.ext?)";
-    WARNING("Potato Gear is off (missing description.ext?)");
+    ["Saving: Potato Gear is disabled"] call BIS_fnc_3DENNotification;
+    WARNING("Potato Gear is disabled");
 };
 
-
 private _problems = [];
 
+
+// Check briefing/author/onLoad attributies:
+private _author = "Scenario" get3DENMissionAttribute "author";
+if (_author == "*** Insert author name here. ***") then {
+    _problems pushBackUnique ["Need to set author", ["Attributes -> General -> Author"]];
+};
+private _slottingInfo = "Intel" get3DENMissionAttribute "IntelOverviewText";
+if (_slottingInfo == "*** Insert mission description here. ***") then {
+    _problems pushBackUnique ["Need to set slotting mission description", ["Attributes -> Multiplayer -> Summary"]];
+} else {
+    if ((parseNumber (_slottingInfo select [0,1])) == 0) then { // very basic ratio detection
```

Comment:
this might not make sense for a COOP, can we parse the mission to check if it's a TvT (assuming it follows mission naming convention) first?

```SQF


private _overviewText = getMissionConfigValue ["overviewText", ""];
if (_overviewText == "*** Insert mission description here. ***") then {
    _problems pushBackUnique ["Major: Need to set slotting mission description", ["Attributes -> Multiplayer -> Summary"]];
};

private _briefNameSplit = (getMissionConfigValue ["briefingName", ""]) splitString "_";
if (count _briefNameSplit < 4) then {
    _problems pushBackUnique ["Major: Mission file name does not follow convention author_typeIdealCount_name_version or is overriden by 3DEN", ["Mission file name"]];
} else {
    private _missionType = toLower (_briefNameSplit select 1);
    if (_missionType find "a" > -1 || _missionType find "tvt" > -1) then {
        if ((parseNumber (_slottingInfo select [0,1])) == 0) then { // very basic ratio detection
            _problems pushBackUnique ["Minor: Should add ratio", ["Attributes -> Multiplayer -> Summary"]];
        };
    };
};

private _onLoadName = [missionConfigFile >> "onLoadName"] call CFUNC(getText);
if (_onLoadName == "" || _onLoadName == "*** Insert mission name here. ***") then {
    _problems pushBackUnique ["Minor: Need to set loading screen info", ["description.ext -> onLoadName"]];
};
```
Does `getMissionConfigValue` work properly at save time?